public/New-DbaAvailabilityGroup.ps1

```diff
@@ -219,6 +227,11 @@ function New-DbaAvailabilityGroup {
 
         Creates a basic availability group named BAG1 on sql2016std and does not confirm when setting up
 
+    .EXAMPLE
+        PS C:\> New-DbaAvailabilityGroup -Primary sql2022n01 -Secondary sql2022n02 -Name AgContained -IsContained
+
+        Creates a contained availability group named AgContained on sql2022
```

Comment:
Instead of "sql2022" (which is your cluster name?) we should name the nodes: "sql2022n01 and sql2022n02".